docs/guides/using-the-network-shim.md

```diff
@@ -149,42 +135,72 @@ This file is meant to be git-ignored and we can use this to set local environmen
 
 _Note that `dhis2Username` and `dhis2Password` are not required directly by the Network Shim, but by the `enableAutologin` setup. They are included in this example since they will be needed for most use-cases._
 
-#### Plugins file
+#### Plugin setup
 
-The `networkShim` plugin should be imported and initialised in the cypress plugins file, see the [cypress docs](https://docs.cypress.io/guides/tooling/plugins-guide#Using-a-plugin) for more info. The plugin function doesn't require the second `config` argument most plugins accept.
+The `networkShim` plugin should be imported and initialised in the
+setupNodeEvents function in the cypress config file (for more information about
+the config file, see [Getting started](../getting-started.md)):
 
 ```js
-const { networkShim } = require('@dhis2/cypress-plugins')
+import { networkShim } from '@dhis2/cypress-plugins'
 
-module.exports = (on, config) => {
-    networkShim(on)
+// ...
+
+async function setupNodeEvents(on, config) {
+    // ...
+
+    networkShim(on, config)
+
+    // ...
 }
+
+// ...
 ```
 
-The `networkShim` plugin accepts a second `options` argument. For most DHIS2 platform apps the default options will be correct so there will be no need to pass a second argument.
+The `networkShim` plugin accepts a second `options` argument. For most DHIS2
+platform apps the options passed to `setupNodeEvents` will be correct so there
+will be no need to pass a second argument.
```

Comment:
I guess this should be *third* not *second* argument right? Also (pretty sure it was me who wrote this, so my apologies), but this text is not very clear at all. I probably meant to say that the _default options_ would be correct...

So this: 
> For most DHIS2 platform apps the options passed to `setupNodeEvents`

Should probably have been:
> For most DHIS2 platform apps the default options of the `networkShim`

Could you also verify that the above is correct? (i.e. that it's a third arg, and that the arg has some default options)

Reply:
> I guess this should be third not second argument right?

Don't think so:
![image](https://user-images.githubusercontent.com/10101831/233245881-74027e87-6ee2-4190-9488-ba802cc1cb0d.png)

> So this:
>
> > For most DHIS2 platform apps the options passed to setupNodeEvents
>
> Should probably have been:
>
> > For most DHIS2 platform apps the default options of the networkShim

Corrected